includes/base/widget-base.php

```diff
@@ -1021,6 +1038,32 @@ public function get_css_config() {
 		return $this->get_widget_css_config( $this->get_group_name() );
 	}
 
+	public function get_responsive_widgets_config() {
+		return [
+			'key' => 'responsive-widgets',
```

Comment:
Put constant in Responsive Widgets Data Manager

Reply:
Done.